app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,79 @@
-.check-your-answers {
 
-  td {
-    @include core-19;
-    vertical-align: top;
+.govuk-check-your-answers {
+
+  > * {
+    position: relative;
+    border-bottom: 1px solid $border-colour;
+
+    > * {
+      display: block;
+    }
   }
 
-  .change-answer {
-    text-align: right;
+  .cya-question {
     font-weight: bold;
-    padding-right: 0;
+    margin: em(12, 19) 4em $gutter/6 0;
```

Comment:
Could we keep the units consistent here and use em(5,19) - rather than the $gutter variable.